code-reviews/recipes/Terraform.md

```diff
@@ -1,18 +1,23 @@
-# Terraform Code Reviews
+## Code Reviews
+### Style Guide
+- Adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html)
+- Ensure all used providers are versioned to prevent breaking changes in future. [docs on versioning providers](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
+- main.tf: This file usually contains provider configuration, backend configuration, imports to the modules to use and eventually some common resources that was not isolated into a specific file. It is absolutely possible to have the whole infrastructure defined in this file itself, suggest to split it into other *.tf files
+- README.md - Always useful to document what the Terraform project does
 
-## Style guide
+### Linters
+1. [tflint](https://github.com/terraform-linters/tflint) - As of now tflint doesn't have Azure specific rules, it can be used to inspect  general Terraform specific issues.  
```

Comment:
I would say something about what tflint does before commenting that there are no specific Azure rules. Not sure why formatting is in Italic. 
NIT: There are two spaces between inspect and general

Reply:
For consistency with other templates, add backticks around extension names, and it would also be good to add some information about installing and using the linter (as in the other templates) -- see the python template as an example

Reply:
@TessFerrandez - updated based on the comment. Added information about installing the TFLint.